osu.Game/Screens/Play/HUD/MultiScoreDiff.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Overlays.SkinEditor;
+using osu.Game.Screens.Select.Leaderboards;
+using osu.Game.Skinning;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public partial class ScoreDifferenceToFirstPlaceInMultiplayDisplay : RollingCounter<long>, ISerialisableDrawable
```

Comment:
`ScoreDifferenceToFirstPlaceInMultiplayerDisplay` if anything. "multiplay" is not a thing. also rename the whole file to match the class.